transcendence-app/src/shared/db/postgres/utils.ts

```diff
@@ -36,6 +36,29 @@ export const makeQuery = async <T>(
   }
 };
 
+export function insertWithClient<T extends Record<string, any>>(
+  client: PoolClient,
+  table: table,
+  entity: T,
+) {
+  const { cols, params, values } = entityQueryMapper(entity);
+  const text = `INSERT INTO ${table}(${cols.join(', ')}) VALUES (${params.join(
+    ',',
+  )}) RETURNING *;`;
+  return client.query(text, values);
+}
+export function updateByIdWithClient<T extends Record<string, any>>(
```

Comment:
Why are these better here than in base postgres repository?

Reply:
I think it's better here because the T type could be any entity, while the base repository is always the same entity.

We could add another type to the base repository <T, U>, but I think it would make it less readable.

Reply:
Check comment above, still think this is not a util

Reply:
Thanks, done in #310